### PR TITLE
Support NUE-ZBFLB

### PR DESCRIPTION
**Pull request by OzGav**
Add support for 3A/Nue Smart Fan Light Switch model NUE-ZBFLB

### CODE REVIEW

**Comment by OzGav** [comment]
I think that addition I have made here is correct although I am having problems sending commands to the switch as per my post here: https://community.home-assistant.io/t/zigbee2mqtt-getting-rid-of-your-proprietary-zigbee-bridges-xiaomi-hue-tradfri/52108/1916

File: devices.js [hunk 1]
```diff
@@ -5374,7 +5374,25 @@ const devices = [
...
+    {
+        zigbeeModel: ['LXN56-1S27LX1.2'],
+        model: 'NUE-ZBFLB',
+        vendor: 'Nue / 3A',
+        description: 'Smart Fan Light Switch',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+        endpoint: (device) => {
+            return {'light': 1, 'fan_high': 2, 'fan_med': 3, 'fan_low': 4};
```
Comment (Koenkk) [inline devices.js:5386]: can you change this to `return {'l1': 1, 'l2': 2, 'l3': 3, 'l4': 4};` and control it via `zigbee2mqtt/Hannah Light Fan Switch/l1/set` payload `{"state": "ON"}`, does this work?

**Comment by OzGav** [comment]
Hi Koen,
I did that and things have changed but not sure for the better? These entities have now appeared:
```
switch.hannah_light_fan_switch_switch_top_left
switch.hannah_light_fan_switch_switch_top_right
switch.hannah_light_fan_switch_switch_bottom_left
switch.hannah_light_fan_switch_switch_bottom_right
```

Other than the friendly name they all have the same attributes:
```
state_light: OFF
linkquality: 23
state_fan_high: OFF
state_fan_med: OFF
state_fan_low: OFF
state_11: ON
friendly_name: Hannah Light Fan Switch switch_top_left
```

I manually switched the light on and I see `state_11:ON` but none of the entities have turned on. 

I used MQTT explorer to send the message and this is what appeared in the log:
```
Zigbee2MQTT:debug 2020-09-03 09:24:44: Received MQTT message on 'zigbee2mqtt/Hannah Light Fan Switch/11/set' with data '{"state": "OFF"}'
Zigbee2MQTT:error 2020-09-03 09:24:44: Failed to call 'EntityPublish' 'onMQTTMessage' (TypeError: Cannot read property 'constructor' of undefined
    at EntityPublish.onMQTTMessage (/zigbee2mqtt-1.14.3/lib/extension/publish.js:173:52)
    at Controller.callExtensionMethod (/zigbee2mqtt-1.14.3/lib/controller.js:375:44)
    at Controller.onMQTTMessage (/zigbee2mqtt-1.14.3/lib/controller.js:265:14)
    at MQTT.emit (events.js:315:20)
    at MQTT.onMessage (/zigbee2mqtt-1.14.3/lib/mqtt.js:95:14)
    at MqttClient.emit (events.js:315:20)
    at MqttClient._handlePublish (/zigbee2mqtt-1.14.3/node_modules/mqtt/lib/client.js:1269:12)
    at MqttClient._handlePacket (/zigbee2mqtt-1.14.3/node_modules/mqtt/lib/client.js:409:12)
    at work (/zigbee2mqtt-1.14.3/node_modules/mqtt/lib/client.js:320:12)
    at Writable.writable._write (/zigbee2mqtt-1.14.3/node_modules/mqtt/lib/client.js:334:5))
```

When I manually turn the light off and on this is what appears in the log now:
```
Zigbee2MQTT:debug 2020-09-03 09:26:37: Received Zigbee message from 'Hannah Light Fan Switch', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-03 09:26:37: MQTT publish: topic 'zigbee2mqtt/Hannah Light Fan Switch', payload '{"state_light":"OFF","linkquality":39,"state_fan_high":"OFF","state_fan_med":"OFF","state_fan_low":"OFF","state_11":"OFF"}'
Zigbee2MQTT:debug 2020-09-03 09:26:37: Received Zigbee message from 'Hannah Light Fan Switch', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-03 09:26:37: MQTT publish: topic 'zigbee2mqtt/Hannah Light Fan Switch', payload '{"state_light":"OFF","linkquality":39,"state_fan_high":"OFF","state_fan_med":"OFF","state_fan_low":"OFF","state_11":"OFF"}'
Zigbee2MQTT:debug 2020-09-03 09:27:13: Received Zigbee message from 'Hannah Light Fan Switch', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-03 09:27:13: MQTT publish: topic 'zigbee2mqtt/Hannah Light Fan Switch', payload '{"state_light":"OFF","linkquality":23,"state_fan_high":"OFF","state_fan_med":"OFF","state_fan_low":"OFF","state_11":"ON"}'
Zigbee2MQTT:debug 2020-09-03 09:27:13: Received Zigbee message from 'Hannah Light Fan Switch', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-03 09:27:13: MQTT publish: topic 'zigbee2mqtt/Hannah Light Fan Switch', payload '{"state_light":"OFF","linkquality":23,"state_fan_high":"OFF","state_fan_med":"OFF","state_fan_low":"OFF","state_11":"ON"}'
```

I am thinking I probably need to clear out all the MQTT messages but wanted to wait on your advice first.

**Comment by OzGav** [comment]
I have tried changing `homeassistant.js ` to

`'NUE-ZBFLB': [
        switchEndpoint('11'), switchEndpoint('fan_high'), switchEndpoint('fan_med'),
        switchEndpoint('fan_low'),
    ],`

and this created a new entity `switch.hannah_light_fan_switch_switch_11` which has a state which reflects the switch state. However I still cannot send commands to the switch. I tried `zigbee2mqtt/Hannah Light Fan Switch/set` with a payload of `{"state_11": "OFF"}` but this just returned an error of:

`Zigbee2MQTT:error 2020-09-04 12:26:14: No converter available for 'state_11' (OFF)`

I am really just fumbling in the dark as I don't understand how this works at a basic level... :-(

**Comment by OzGav** [comment]
OK I have gone back to:
 ```
        {
        zigbeeModel: ['LXN56-1S27LX1.2'],
        model: 'NUE-ZBFLB',
        vendor: 'Nue / 3A',
        description: 'Smart Fan Light Switch',
        supports: 'on/off',
        fromZigbee: [fz.on_off],
        toZigbee: [tz.on_off],
        endpoint: (device) => {
            return {'light': 1, 'fan_high': 2, 'fan_med': 3, 'fan_low': 4};
        },
        meta: {configureKey: 1, multiEndpoint: true},
        configure: async (device, coordinatorEndpoint) => {
            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
            await bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
            await bind(device.getEndpoint(4), coordinatorEndpoint, ['genOnOff']);

        },
    },
```

and `homeassistant.js`
```
    'NUE-ZBFLB': [
        switchEndpoint('light'), switchEndpoint('fan_high'), switchEndpoint('fan_med'),
        switchEndpoint('fan_low'),
    ],
```

and I have four entities again (eg. switch.hannah_light_fan_switch_switch_light ) and what I have found is that if I send a message from MQTT explorer of `zigbee2mqtt/Hannah Light Fan Switch/set` with payload `{"state": "OFF"}` then the light will turn off (and on depending on the payload). However I cannot make any change from home assistant because if I flip the switch I see in the log

`Zigbee2MQTT:error 2020-09-04 13:12:50: Entity 'Hannah Light Fan Switch/light' is unknown`

As an example looking at MQTT explorer i can see the command_topic is `zigbee2mqtt/Hannah Light Fan Switch/light/set` but if I try and publish to that I get that error message so I assuming it just a configuration thing I need to get right in zigbee2mqtt? Here is the contents of MQTT explorer for `homeassistant/switch/0xec1bbdfffed0f6ee`

```
{
"availability_topic":"zigbee2mqtt/bridge/state",
"command_topic":"zigbee2mqtt/Hannah Light Fan Switch/light/set",
"device":{
  "identifiers":[
    "zigbee2mqtt_0xec1bbdfffed0f6ee"
  ],
  "manufacturer":"Nue / 3A",
  "model":"Smart Fan Light Switch (NUE-ZBFLB)",
  "name":"Hannah Light Fan Switch",
  "sw_version":"Zigbee2MQTT 1.14.4"
},
"json_attributes_topic":"zigbee2mqtt/Hannah Light Fan Switch",
"name":"Hannah Light Fan Switch switch_light",
"payload_off":"OFF",
"payload_on":"ON",
"state_topic":"zigbee2mqtt/Hannah Light Fan Switch",
"unique_id":"0xec1bbdfffed0f6ee_switch_light_zigbee2mqtt",
"value_template":"{{ value_json.state_light }}"
}
```

**Comment by Koenkk** [comment]
- Can you provide the `data/database.db` entry of this device?
- Can you control it through the topic/payload I provided here: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1520#discussion_r482398498 ?

**Comment by OzGav** [comment]
Do you mean this entry?
`{"id":17,"type":"Router","ieeeAddr":"0xec1bbdfffed0f6ee","nwkAddr":35287,"manufId":4098,"manufName":"3A Smart Home DE","powerSource":"Unknown","modelId":"LXN56-1S27LX1.2","epList":[1,2,3,4],"endpoints":{"1":{"profId":260,"epId":1,"devId":0,"inClusterList":[0,3,4,5,6,8,4096],"outClusterList":[4096],"clusters":{"genBasic":{"attributes":{"manufacturerName":"3A Smart Home DE","powerSource":0,"zclVersion":3,"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"","swBuildId":""}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b0014d92e46","endpointID":1}]},"2":{"profId":260,"epId":2,"devId":0,"inClusterList":[0,3,4,5,6,8,4096],"outClusterList":[4096],"clusters":{"genBasic":{"attributes":{"modelId":"LXN56-1S27LX1.2","appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"","swBuildId":""}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b0014d92e46","endpointID":1}]},"3":{"profId":260,"epId":3,"devId":0,"inClusterList":[0,3,4,5,6,8,4096],"outClusterList":[4096],"clusters":{"genBasic":{"attributes":{"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"","swBuildId":""}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b0014d92e46","endpointID":1}]},"4":{"profId":260,"epId":4,"devId":0,"inClusterList":[0,3,4,5,6,8,4096],"outClusterList":[4096],"clusters":{"genBasic":{"attributes":{"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"","swBuildId":""}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b0014d92e46","endpointID":1}]}},"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"","swBuildId":"","zclVersion":3,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1598930413864}`

When I posted that MQTT message you suyggested from MQTT Explorer (after changing the words to numbers)  I saw this in the log

```
Zigbee2MQTT:debug 2020-09-05 08:43:38: Received MQTT message on 'zigbee2mqtt/Hannah Light Fan Switch/l1/set' with data '{"state": "ON"}'
Zigbee2MQTT:error 2020-09-05 08:43:40: Failed to call 'EntityPublish' 'onMQTTMessage' (AssertionError [ERR_ASSERTION]: Endpoint name 'l1' is given but device has no such endpoint
    at Zigbee.resolveEntity (/zigbee2mqtt-1.14.4/lib/zigbee.js:214:25)
    at EntityPublish.onMQTTMessage (/zigbee2mqtt-1.14.4/lib/extension/publish.js:64:44)
    at Controller.callExtensionMethod (/zigbee2mqtt-1.14.4/lib/controller.js:365:44)
    at Controller.onMQTTMessage (/zigbee2mqtt-1.14.4/lib/controller.js:262:14)
    at MQTT.emit (events.js:315:20)
    at MQTT.onMessage (/zigbee2mqtt-1.14.4/lib/mqtt.js:95:14)
    at MqttClient.emit (events.js:315:20)
    at MqttClient._handlePublish (/zigbee2mqtt-1.14.4/node_modules/mqtt/lib/client.js:1271:12)
    at MqttClient._handlePacket (/zigbee2mqtt-1.14.4/node_modules/mqtt/lib/client.js:410:12)
    at work (/zigbee2mqtt-1.14.4/node_modules/mqtt/lib/client.js:321:12))
```

I have gone back to my original entry and if I post to `zigbee2mqtt/Hannah Light Fan Switch/light/set` with payload `{"state": "ON"}` I see in the log

```
Zigbee2MQTT:debug 2020-09-05 08:48:35: Received MQTT message on 'zigbee2mqtt/Hannah Light Fan Switch/light/set' with data '{"state": "ON"}'
Zigbee2MQTT:info  2020-09-05 08:48:35: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"Hannah Light Fan Switch/light"},"type":"entity_not_found"}'
Zigbee2MQTT:error 2020-09-05 08:48:35: Entity 'Hannah Light Fan Switch/light' is unknown
```

**Comment by Koenkk** [comment]
Only endpoint names defined here can be used: https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/util/utils.js#L31 (so `light` cannot be used).

**Comment by OzGav** [comment]
AAAHHHHhhhhhh. Important information :-) I have this working now but I added my names to utils.js. What do you think of adding a pull request to add my endpoint names? If we can't do that then I will make some changes here to make it work with the existing names.

**Comment by Koenkk** [comment]
Preferable not, as the endpoint names cannot be used in the `friendly_name` (so e.g. `friendly_name: kitchen/light` wouldn't be possible anymore)

**Comment by Koenkk** [comment]
Note to self, after this merge https://github.com/Koenkk/zigbee2mqtt/pull/4279

**Comment by OzGav** [comment]
OK what about changing it to light_button, fan_high_button etc...?

**Comment by Koenkk** [comment]
e.g. `button_light` is fine

**Comment by OzGav** [comment]
OK I believe this is now working. However I do notice one thing and have one question. Since this last change I notice that the attributes now longer contain an entry for all of the switch states, it only has an entry for the last state that changed. As I said everything is working but I just wanted to point that out in case it is important. 

My question is: there are attributes for all the other entries that I have tried and I would like to tidy that up and remove them. Do you know where this retained information is stored? All four switches have these attributes which are identical except for the friendly name. You will notice an attribute for `state_button_light` as that is the last one that was changed. All the rest never change now.

`
linkquality: 63

state: OFF
state_11: ON
state_12: OFF
state_13: OFF
state_14: OFF
state_button_light: OFF
state_fan_high: OFF
state_fan_low: OFF
state_fan_med: OFF
state_light: OFF
friendly_name: Hannah Light Fan switch_button_fan_low
`

**Comment by Koenkk** [comment]
> OK I believe this is now working. However I do notice one thing and have one question. Since this last change I notice that the attributes no longer contain an entry for all of the switch states, it only has an entry for the last state that changed. As I said everything is working but I just wanted to point that out in case it is important.

I don't understand this, can you explain a bit more?

> My question is: there are attributes for all the other entries that I have tried and I would like to tidy that up and remove them. Do you know where this retained information is stored? 

In `data/state.json` (just remove the whole entry for the device there)

**Comment by OzGav** [comment]
I am running on a RPi so I found state.json in share/zigbee2mqtt I trust that is the correct one. In any case I found the entry and deleted it, cleared the topic using MQTT explorer (under both homeassistant and zigbee2mqtt) but all of the attributes just came back when I toggled a switch?

On the odd attribute behaviour. The switch is working 100% in homeassistant it is just the attributes are odd. Apart from the old attributes I can't remove I expected that I would now see all four of:

```
state_button_light: OFF
state_button_fan_high: OFF
state_button_fan_med: OFF
state_button_fan_low: OFF
```

(because that is what happened with the old attributes) but I don't. I see all the old attributes plus an entry for the last button that was pressed. These attributes are identical in all four switch entities (ie. `sensor.hannah_light_fan_linkquality`, `switch.hannah_light_fan_switch_button_fan_high`, `switch.hannah_light_fan_switch_button_fan_med`, `switch.hannah_light_fan_switch_button_fan_low` and `switch.hannah_light_fan_switch_button_light`) except for the friendly name. So if I turn on the light I see:
```
linkquality: 39
state: OFF
state_11: ON
state_12: OFF
state_13: OFF
state_14: OFF
state_button_light: ON
state_fan_high: OFF
state_fan_low: OFF
state_fan_med: OFF
state_light: OFF
friendly_name: Hannah Light Fan switch_button_fan_high
```
and then if I press the fan low button I see:
```
linkquality: 39
state: OFF
state_11: ON
state_12: OFF
state_13: OFF
state_14: OFF
state_button_fan_low: ON
state_fan_high: OFF
state_fan_low: OFF
state_fan_med: OFF
state_light: OFF
friendly_name: Hannah Light Fan switch_button_fan_high
```
Note the only change is the line below state_14. Again all five entities all always showing exactly the same thing except for the friendly name. This doesn't seem to be affecting functionality but is unusual. Thoughts?

**Comment by Koenkk** [comment]
When clear `state.json`, did you first stop zigbee2mqtt, clear `state.json` and start zigbee2mqtt?

**Comment by OzGav** [comment]
No I didn't but now I have that is fixed :-) Again everything is working 100% just the attributes only show this (for example):
```
linkquality: 81
state_button_fan_high: OFF
friendly_name: Hannah Light Fan switch_button_light
```
All entities show the same thing except for the friendly name and that one state attibute just changes depending on what was the last button to change state. Doesn't seem to be a problem so if you are happy I think this is ready to merge.

**Comment by Koenkk** [comment]
- I expect that all `state_button_*` should be in if you controlled all buttons at least once. Can you provide me the debug logging of when controlling all buttons at least once?

To enable debug logging set in `configuration.yaml`:
```yaml
advanced:
  log_level: debug
```

- Question: when you set e.g. `state_button_fan_high` to `ON` and then set `state_button_fan_med` to ON, does `state_button_fan_high` automatically go to `OFF`?

**Comment by OzGav** [comment]
In answer to your question: YES

Here is button LIGHT - LOW - MED - HIGH - HIGH - LIGHT
```
Zigbee2MQTT:debug 2020-09-10 23:34:41: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:41: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":84,"state_button_light":"ON"}'
Zigbee2MQTT:debug 2020-09-10 23:34:41: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:41: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":84,"state_button_light":"ON"}'
Zigbee2MQTT:debug 2020-09-10 23:34:42: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 4 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:42: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":76,"state_button_fan_low":"ON"}'
Zigbee2MQTT:debug 2020-09-10 23:34:44: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 3 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:44: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":78,"state_button_fan_med":"ON"}'
Zigbee2MQTT:debug 2020-09-10 23:34:44: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 4 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:44: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":78,"state_button_fan_low":"OFF"}'
Zigbee2MQTT:debug 2020-09-10 23:34:45: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 2 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:45: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":73,"state_button_fan_high":"ON"}'
Zigbee2MQTT:debug 2020-09-10 23:34:45: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 3 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:45: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":73,"state_button_fan_med":"OFF"}'
Zigbee2MQTT:debug 2020-09-10 23:34:46: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 2 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:46: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":73,"state_button_fan_high":"OFF"}'
Zigbee2MQTT:debug 2020-09-10 23:34:47: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:47: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":84,"state_button_light":"OFF"}'
Zigbee2MQTT:debug 2020-09-10 23:34:47: Received Zigbee message from 'SpareBed Light Fan', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2020-09-10 23:34:47: MQTT publish: topic 'zigbee2mqtt/SpareBed Light Fan', payload '{"linkquality":84,"state_button_light":"OFF"}'
```

**Comment by Koenkk** [comment]
Can you share me your `configuration.yaml`?

**Comment by OzGav** [comment]
This is share/zigbee2mqtt/configuration.yaml
```
{
  "devices": "devices.yaml",
  "groups": "groups.yaml",
  "homeassistant": true,
  "permit_join": false,
  "mqtt": {
    "base_topic": "zigbee2mqtt",
    "server": "mqtt://core-mosquitto",
    "user": "USER",
    "password": "PASSWORD"
  },
  "serial": {
    "port": "/dev/ttyACM0"
  },
  "advanced": {
    "log_level": "debug",
    "pan_id": 6754,
    "channel": 14,
    "network_key": [
      1,
      3,
      5,
      7,
      9,
      11,
      13,
      15,
      0,
      2,
      4,
      6,
      8,
      10,
      12,
      13
    ],
    "availability_blacklist": []
  },
  "ban": [],
  "whitelist": [],
  "queue": {}
}
```
I notice that in the actual config screen of the addon I also have these additional lines at the bottom:
```
socat:
  enabled: false
  master: 'pty,raw,echo=0,link=/dev/ttyZ2M,mode=777'
  slave: 'tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5'
  restartdelay: 1
  initialdelay: 1
  options: '-d -d'
  log: false
discovery: true
zigbee_shepherd_devices: true
```

**Comment by Koenkk** [comment]
Does the `state.json` contain the other `state` values?

**Comment by OzGav** [comment]
No. It currently just has this:
`"0xec1bbdfffed0f6ee":{"linkquality":55}`

**Pull request merged**